go/vt/vttablet/endtoend/misc_test.go

```diff
@@ -304,7 +304,7 @@ func TestConsolidation(t *testing.T) {
 	framework.Server.SetPoolSize(1)
 
 	for sleep := 0.1; sleep < 10.0; sleep *= 2 {
-		query := fmt.Sprintf("select sleep(%v) from dual", sleep)
+		query := fmt.Sprintf("select sleep(%v) from dual /* query: %v */", sleep, sleep)
```

Comment:
I think we need to tweak this test a bit. If we do it like this, is not really testing with different comments. Something like this will do:
```
		vstart := framework.DebugVars()
		var wg sync.WaitGroup
		wg.Add(2)
		go func() {
			query := fmt.Sprintf("select sleep(%v) from dual /* query: 1 */", sleep)
			framework.NewClient().Execute(query, nil)
			wg.Done()
		}()
		go func() {
			query := fmt.Sprintf("select sleep(%v) from dual /* query: 2 */", sleep)
			framework.NewClient().Execute(query, nil)
			wg.Done()
		}()
```